evolution-chain/precompile/laos-evolution/contracts/LaosEvolution.sol

```diff
@@ -16,4 +16,12 @@ interface LivingAssets {
     /// @param owner the owner of the newly created collection
     /// @return collectionId the id of the newly created collection
     function createCollection(address owner) external returns (uint64);
+
+    /// @notice Owner of the collection
+    /// @dev Call this function to get the owner of a collection
+    /// @param collectionId the id of the collection
+    /// @return owner the owner of the collection
```

Comment:
If the return of the method does not have an owner, then you don't need to use it in the @return.
In this case, you should use
/// @return the owner of the collection

instead, if you had this: function ownerOfCollection(collectionId) returns (address admin);

you would have to use:
/// @return admin the owner of the collection

Reply:
And indeed, the same applies to line 17, you should just use 

    /// @return the id of the newly created collection